test/filter_test.dart

```diff
@@ -263,5 +263,42 @@ void main() {
             'All'
           ]..sort()));
     });
+
+    test('findNodeByPath no input', () {
```

Comment:
Rename the tests too:
```suggestion
    test('findNodesByPath no input', () {
```